handbook/engineering/leadership/index.md

```diff
@@ -43,33 +43,33 @@ We have a 1hr sync every Monday at 9:30am PT.
 - Write one sentence that highlights one important thing (for example: changes in important metrics, customer feedback collected, things shipped) the team made progress on this week and include a image/screenshot when possible.
   - The sentence should articulate what specifically was completed and why it was important.
     - Processes like planning, retrospectives, offsites, hiring, and post-mortems are important and valuable ingredients to generating progress over time, but are not themselves progress, so shouldn't be highlighted here.
     - This progress should be framed in a way that is understandable by anyone at the company.
   - Great highlights will be presented at company meeting by the VP of Eng or VP of Product.
     - It is helpful if screenshots are somewhere between squarish or 4:3. Very wide screenshots will not scale well on a slide.
     - If you can, include a link to where you took a screenshot so we can retake the screenshot at a different display ratio if necessary.
-- Answer the following questions for each goal that your team is working on:
-  - What important things happened this week (for example: changes in important metrics, customer feedback collected, things shipped)?
-  - What are the plans for next week?
-  - What should my peer-team and manager be aware of?
 
-Good examples:
+### Status updates
+
+The goal of status updates is to communicate each team's progress and tie that progress to larger company goals.
+
+Engineering managers send an update to [engineering-team-status@sourcegraph.com](https://groups.google.com/a/sourcegraph.com/g/engineering-team-status) after each iteration retrospective. Teams have different iteration cadences and as such we do not expect updates to come through on the same day, but it should follow soon after you have completed your retrospective for the iteration. 
+
+Delegate to someone else on your team to send the update when you are on vacation.
+
+Anyone at the company can join the group to subscribe to these updates, but your primary audience is your manager (VP Eng and VP Product) and your peers (EMs/PMs). You can also cc your own team.
+
+Please include the following content in your update:
 
-- https://groups.google.com/a/sourcegraph.com/g/engineering-team-status/c/7zHism3B44w/m/mK2mT4AaAgAJ
-- https://groups.google.com/a/sourcegraph.com/g/engineering-team-status/c/6zTRJu2JYc0/m/VCOPNVwbAgAJ
+- The subject of the email should contain the team name and the date the update is being sent (e.g. "Search update 2020-10-14") so each update starts a distinct email thread (otherwise they get grouped together in the Google Groups UI).
+- Follow the [3Ps](https://en.wikipedia.org/wiki/Progress,_plans,_problems) approach in reporting on the Progress, Problems and Plans against the team's goals.
```

Comment:
```suggestion
- Follow the [3Ps](https://en.wikipedia.org/wiki/Progress,_plans,_problems) approach in reporting on the Progress, Concerns and Plans against the team's goals.
```

I think we agreed to use Concerns instead of problems.

Reply:
In Monday meeting, I heard the word "Challenges" (which would be my preference), but I don't know if another convo happened in the EM team sync.